lib/deployinator/templates/layout.mustache

```diff
@@ -456,6 +456,9 @@ _  __  / _  _ \___  __ \__  / _  __ \__  / / /__  / __  __ \_  __ `/_  __/_  __
                                             addStackLockStatus(data.method, data.who, data.lock_time);
                                         }
                                     }
+                                },
+                                error: function (xhrObj, textStatus, errorThrown) {
```

Comment:
i noticed that other places in this file the callback function for error: is just function(data) , here it uses three parameters -  in either case none of the parameters ever seem to be used - should we change this to look like the others for posterity?